### PR TITLE
INT-1287 and INT-1337

### PR DESCRIPTION
**Pull request by ahaias-tc**
I have combined INT-1287 and INT-1337 in a single PR.

### CODE REVIEW

**Review by sarus — CHANGES_REQUESTED** [review]
@ahaias-tc Functionality works great.  Left some comments in various places.  Please let me know if you have any questions!

File: templates/block.hbs [hunk 1]
```diff
@@ -1,436 +1,414 @@
...
+  </div>
+  {{#each newIocsToSubmit as |entity|}}
+  {{#if (eq entity.displayedType "domain")}}
+  <h1 class="p-title submission-options py-1">
+    {{fa-icon "filter" size="sm" fixedWidth=true}} Host Specific Filters
+  </h1>
+  <div class="checkbox-container">
+    <label>
+      <input type="checkbox" {{on "change" (action "updateWhoIsValue" )}} disabled={{interactionDisabled}} />
+      <span class="p-key">WhoIs</span>
```
[the remark this answers is not in the document]
Comment (ahaias-tc) [reply templates/block.hbs:237]: Might be some differences between Mac and Windows? It looks aligned on my side, but I will definitely try the change you suggested. The reason I am asking this is that I specifically added some styling to make it aligned.

[the remark this answers is not in the document]
Comment (sarus) [reply templates/block.hbs:237]: @ahaias-tc Could definitely be a OS difference.  We've even seen differences in Chrome across OSes when it comes to CSS styling.  Are you using Safari to test?  At least on Windows, it seems like the label block element also needs to have the flex box styling to center align the text and checkbox (not just the parent container).

[the remark this answers is not in the document]
Comment (ahaias-tc) [reply templates/block.hbs:237]: No, I am testing on Chrome. I hate Safari

**Review by sarus — CHANGES_REQUESTED** [review]
@ahaias-tc The package-lock has conflicts and is preventing merging.  Since no changes were made to the package.json I would revert the package-lock the version in the `develop` branch to resolve the conflict.

Otherwise changes look good!

**Review by sarus — CHANGES_REQUESTED** [review]
@ahaias-tc Sorry for all the back and forth but the integration development checklist isn't passing.  To merge into develop you'll need to bump the version in the `package.json` file (and possible modify the version in the package-lock as well).

Since you added a new feature I would suggest bumping the version to `3.1.0`

**Review by sarus — APPROVED** [review]
@ahaias-tc Looks great!

**Pull request merged**